### PR TITLE
Adding the options borderless and maximized to win

### PR DESCRIPTION
**Pull request by Id405**
This pull request adds two options to the win package:

1. Borderless this makes the window have no top bar and sets the hint GLFW_DECORATED to GLFW_FALSE
2. Maximized which, when true makes the window start maximized using the hint GLFW_MAXIMIZED at GLFW_TRUE

This pr also adds a fix for w.ratio being 0

### CODE REVIEW

File: win/win.go [hunk 1]
```diff
@@ -44,6 +46,20 @@ func Resizable() Option {
 	}
 }
 
+//Borderless option makes the window borderless
```
Comment (faiface) [inline win/win.go]: Space after // and period at the end of the sentence.

File: win/win.go [hunk 2]
```diff
@@ -44,6 +46,20 @@ func Resizable() Option {
...
+func Borderless() Option {
+	return func(o *options) {
+		o.borderless = true
+	}
+}
+
+//Maximized option makes the window start maximized
```
Comment (faiface) [inline win/win.go]: Ditto.

File: win/win.go [hunk 3]
```diff
@@ -53,6 +69,8 @@ func New(opts ...Option) (*Win, error) {
 		width:     640,
 		height:    480,
 		resizable: false,
+		borderless: false,
```
Comment (faiface) [inline win/win.go]: Use gofmt.

**Comment by Id405** [comment]
No problem, I think I approved the changes, however I dont fully understand github.

**Comment by faiface** [comment]
Yeah, you don't, but that's okay :D. Those weren't changes, but requirements. Now it's your job to add new commits that resolve those requirements.

**Comment by Id405** [comment]
Oh, makes sense. Ill make the changes as soon as I get home tonight

**Comment by faiface** [comment]
Ping

**Comment by Id405** [comment]
That should have fixed it.

**Comment by faiface** [comment]
Alright, this is good now! Thanks a lot for this!

**Pull request merged**